.irbrc

```diff
@@ -1 +1,5 @@
 IRB.conf[:USE_AUTOCOMPLETE] = false
+
+on_deployed_box = File.directory?('/srv/idp/releases/')
+
+IRB.conf[:SAVE_HISTORY] = on_deployed_box ? nil : 1000
```

Comment:
we could put `0` here because it gets `#to_i`'d anyways, but my thought was maybe this was clearer